arlas-rest/src/main/java/io/arlas/server/rest/explore/describe/DescribeCollectionRESTService.java

```diff
@@ -51,39 +54,45 @@ public DescribeCollectionRESTService(ExploreService exploreService) {
     @GET
     @Produces(UTF8JSON)
     @Consumes(UTF8JSON)
-    @ApiOperation(value = "Describe", produces = UTF8JSON, notes = "Describe the structure and the content of the given collection. ", consumes = UTF8JSON, response = CollectionReferenceDescription.class)
-    @ApiResponses(value = {@ApiResponse(code = 200, message = "Successful operation", response = CollectionReferenceDescription.class, responseContainer = "CollectionReferenceDescription"),
-            @ApiResponse(code = 500, message = "Arlas Server Error.", response = Error.class),
-            @ApiResponse(code = 403, message = "Collection not authorized.", response = Error.class),
-            @ApiResponse(code = 400, message = "Bad request.", response = Error.class)})
+    @Operation(summary = "Describe",  description = "Describe the structure and the content of the given collection.")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = @Content(array = @ArraySchema(schema = @Schema(implementation = CollectionReferenceDescription.class)))),
```

Comment:
it shoudn't return an array but onl an object

Reply:
Done